src/apis/group/group.service.js

```diff
@@ -207,6 +207,33 @@ class GroupService {
         });
     }
 
+    /**
+     * getting all users based on groupId 
+     */
+
+    getAllUserNamesByGroupId(groupId, callback) {
```

Comment:
Rename the method name to `getAllUsersByGroupId` and return the users instead of just names.